niksynov/stfalcon-dev-task
# Added Ticket entity and ticket workflow, to be able to mark ticket as paid and send receipt or mark it as canceled

## How to test

1. Run docker composer up: `docker-compose up -d`
2. Check docker container id or name `docker ps`
3. Enter docker container: `docker exec -it {id} bash`
4. Run `vendor/bin/phpunit`

There is one kernel test that runs the state resolver for the ticket entity. After the ticket entity is marked as paid, it triggers a subscriber that calls the receipt sender, which will send a receipt notification to the user. Alternatively, if a ticket is canceled, it will just move to the canceled state.
